static/src/js/components/CollectionSelector/CollectionSelector.scss

```diff
@@ -0,0 +1,23 @@
+@use "sass:map";
+
+@import '../../../css/vendor/bootstrap/variables';
+
+.collection-selector {
+  &__list-group {
+    height:31.25rem;
+  }
+
+  &__list-group-item {
+    cursor: pointer;
+  }
+
+  &__list-group-item-selected {
```

Comment:
Last thing. These modifier classes should be written like `--is-selected` or `--is-available`.

You can do something like this with the scss:

```css
&__list-group-item {
    ...

    &--is-selected {
        ...
    }
    
    &--is-available {
        ...
    }
}
```